docs/privacy-policy.md

```diff
@@ -0,0 +1,127 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to gathering NO personal information through sempervirens. We are 
+committed to being transparent and open. This Privacy Policy explains generically how 
+we receive information from you, and what we do with that  information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information."  Such combinations will not be collected. Removing all 
+personal information creates a data set that is deemed "non-personal information" 
+and may be collected.
+
+How do we learn information about you?
+---------------------------------------
+
+We learn non-personal information about you when:
+
+* you give it to us directly (e.g., when you choose to send us crash reports);
+* we collect it automatically through our products and services (e.g., the Sempervirens 
+  code); or
+* when you have opted-out of sending other non-personal information, we record that an 
+  opt-out event has occurred.
+
+
+What do we do with your information once we have it?
+----------------------------------------------------
+
+When you give us information, we will use it in the ways for which you've given us 
+permission. Generally, we use your information to help us provide and improve our 
+products and services to you and for you. This includes aggregations of the information 
+to display data such as global usage statistics.
+
+What are usage statistics?
+--------------------------
+
+Usage statistics or "telemetry" is a feature of Sempervirens that sends Numfocus usage, 
+performance, and responsiveness statistics about user interface features, application 
+program interfaces (APIs), application binary interfaces (ABIs), memory, hardware 
+configuration. Your IP address is also collected as a part of a standard web log. 
```

Comment:
Sorry, I mean uuid.uuid4(). (The one that generates a random uuid.)
On Jul 16, 2015 2:12 PM, "Nathaniel Smith" <njs@pobox.com> wrote:

> uuid.uuid() seems as good as anything.
> On Jul 16, 2015 2:08 PM, "Anthony Scopatz" <notifications@github.com>
> wrote:
>
>> In docs/privacy-policy.md
>> <https://github.com/njsmith/sempervirens/pull/16#discussion_r34838092>:
>>
>> > +
>> > +What do we do with your information once we have it?
>> > +----------------------------------------------------
>> > +
>> > +When you give us information, we will use it in the ways for which you've given us
>> > +permission. Generally, we use your information to help us provide and improve our
>> > +products and services to you and for you. This includes aggregations of the information
>> > +to display data such as global usage statistics.
>> > +
>> > +What are usage statistics?
>> > +--------------------------
>> > +
>> > +Usage statistics or "telemetry" is a feature of Sempervirens that sends Numfocus usage,
>> > +performance, and responsiveness statistics about user interface features, application
>> > +program interfaces (APIs), application binary interfaces (ABIs), memory, hardware
>> > +configuration. Your IP address is also collected as a part of a standard web log.
>>
>> @takluyver <https://github.com/takluyver> et al, how is the unique
>> installation ID going to be generated (random UUID)? I need this info to
>> keep writing this up.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/njsmith/sempervirens/pull/16/files#r34838092>.
>>
>